doc/html/_sources/libnvme.rst.txt

```diff
@@ -5245,8 +5245,8 @@ The nvme command status if a response was received (see
     __u16 control;
     __u16 apptag;
     __u16 appmask;
+    __u16 dspec;
     __u8 dsm;
-    __u8 dspec;
```

Comment:
BTW, could you drop the generated documentation update in this patch? This makes backporting unnecessary complex. And the plan is to update the generated documentation right before each release.

Reply:
Thanks for letting me know, I will remove doc updates :)